### PR TITLE
Enable creating/updating the table gpcrondump_history by default for …

### PR DESCRIPTION
**Pull request by royc1**
…each run of gpcrondump.

Add -H option to disable update gpcrondump history table.
Option '-h' will not be deleted in this release, but it will be deprecated in a future release.

Update behave tests and unit tests.

Authors: Chumki Roy, Marbin Tan, Nikhil Kak

### CODE REVIEW

File: gpMgmt/bin/gppylib/test/behave/mgmt_utils/backup.feature [hunk 1]
```diff
@@ -1168,22 +1168,31 @@ Feature: Validate command line arguments
...
         And verify that the table "gpcrondump_history" in "bkdb" has dump info for the stored timestamp
-        When the user runs "gpcrondump -a -x bkdb -h --incremental"
+        When the user runs "gpcrondump -a -x bkdb --incremental"
         And the timestamp from gpcrondump is stored
         Then gpcrondump should return a return code of 0
         And verify that there is a "heap" table "gpcrondump_history" in "bkdb"
+        And verify that table "gpcrondump_history" in "bkdb" has "2" rows
         And verify that the table "gpcrondump_history" in "bkdb" has dump info for the stored timestamp
 
+    Scenario: Verify the gpcrondump -h option should not create history table
```
Comment (k-huddleston) [inline gpMgmt/bin/gppylib/test/behave/mgmt_utils/backup.feature]: Should this be capital H (-H) in the scenario title?

**Comment by k-huddleston** [comment]
We left a comment, but otherwise looks good :)

**Comment by Chibin** [comment]
Does the gpcrondump help file need to be updated?

**Comment by royc1** [comment]
Pushed.
Thanks for review.

**Pull request closed without merging**